server.js

```diff
@@ -27,11 +22,105 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+
+if (process.env.RESET_DB) {
+  console.log('Database reset')
+  const seedDatabase = async () => {
+    await Author.deleteMany({})
+    await Book.deleteMany({})
+
+    // creating array with unique Authors from data
+    const uniqueAuthors = [... new Set(booksData.map(book => book.authors))]
```

Comment:
This is a really good find to prevent duplicate authors from being present in your seed data. Nice